hyco-https/tests/chunkedpost.test.js

```diff
@@ -1,18 +1,22 @@
 var https = require('..')
 
-jest.setTimeout(30000); // 30 seconds
+var totalRequests = 100; // Total requests to send over the test
```

Comment:
>totalRequests [](start = 4, length = 13)

How long does the test take with 100 requests? Still pretty quick?

Reply:
It would be nice if each of these tests finished in 5 seconds or less apiece.

---
In reply to: [315833653](https://github.com/Azure/azure-relay-node/pull/32#discussion_r315833653) [](ancestors = 315833653)

Reply:
I have run the tests around 10 times each, it takes anywhere from 2-15 seconds to finish 100 requests

---
In reply to: [315834909](https://github.com/Azure/azure-relay-node/pull/32#discussion_r315834909) [](ancestors = 315834909,315833653)